app/services/VersionService.scala

```diff
@@ -131,6 +131,12 @@ class VersionService @Inject()(configuration: Configuration,
         r1.plugins ++ r2.plugins
       ))
       .filter(repo => repo.dependencies.nonEmpty || repo.plugins.nonEmpty)
+      .map(r => Repository(
```

Comment:
With scala `case class` when you want to change only some fields it's better to use the `copy` method (the same principle is taken up by the kotlin `data class`)
```
      .map(r => r.copy(
        dependencies = r.dependencies.sortBy(d => (d.subfolder, d.getType, d.name)),
        plugins = r.plugins.sortBy(p => (p.getType, p.name)))
      )
```

Reply:
OK, I've updated the `reduceOption()` call as well using the same construct.